canonical-web-and-design/charmhub.io
# Add webpack

## Done
- Add webpack and temporary modules

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8045
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Visit http://0.0.0.0:8045/ the global nav should still load
- Visit http://0.0.0.0:8045/store with the console open, you should see a message
- Visit http://0.0.0.0:8045/wordpress with the console open, you should see a message
- All the checks on this PR should pass

## Issue / Card

Fixes #80
